.github/workflows/rspec.yml

```
@@ -0,0 +1,32 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    environment: test
+    container:
+      image: ruby:3.0.6
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Gem cache
+        id: cache-bundle
+        uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: bundle-${{ hashFiles('**/Gemfile.lock') }}
+
+      - name: Bundle install
+        env:
+          RAILS_ENV: test
+        run: |
+          bundle install
+      - name: Run tests
+        env:
```

🎨 Just to keep the steps equally spaced in the file

```suggestion
        run: |
          bundle install
          
      - name: Run tests
        env:
```